web/src/tab-summary.ts

```diff
@@ -39,11 +41,37 @@ export class TabSummary extends LitElement {
           </div>
         </div>
       </div>
+      ${this.info?.failuresSummary !== undefined ? html `<div class="dropdown-container">
```

Comment:
would be great to ember the table element you defined below here

Reply:
done